.changes/v3.11.0/1129-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix Issue [#1127](https://github.com/vmware/terraform-provider-vcd/issues/1127) (Incorrect behavior of vcd_resource_list, which can retrieve edge gateways belonging to a VDC, but not belonging to a VDC group) [GH-1129]
```

Comment:
Nit. In this context, capitalizing keywords would be helpful:

```suggestion
* Fix Issue [#1127](https://github.com/vmware/terraform-provider-vcd/issues/1127) (Incorrect behavior of vcd_resource_list, which can retrieve Edge Gateways belonging to a VDC, but not belonging to a VDC Group) [GH-1129]
```

Reply:
done